### PR TITLE
Add more companies

### PR DESCRIPTION
**Pull request by davidchoo12**
I have added a few more entries into the table from my own research. I might add more when I found more, either in this PR or a new PR if this one is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,19 @@ To contribute:
 | [JPMorgan](https://jpmchase.taleo.net/careersection/10140/jobdetail.ftl?job=190064684)| | Summer|October 27, 2019|
 | [Jump Trading](https://www.jumptrading.com/jobs.html)  | Downtown | Summer | |
 | [Lazada](http://www.lazada.com/work-at-lazada) | Tanjong Pagar | Summer | |
+| [Microsoft](https://careers.microsoft.com/us/en/job/697177/2020-Summer-Internship-Opportunities-Singapore) | #13-01 Frasers Tower | Summer | Deadline: 1 Dec 2019 |
```
Comment (sushinoya) [inline README.md]: This sounds a lot more like an internship geared towards MBA students. They seem to be open to Bachelor's too but the description and requirements do not suggest that the applicant will be working on anything technical.

Comment (davidchoo12) [reply README.md]: Oops, you're right. I'll remove it.

File: README.md [hunk 2]
```diff
@@ -30,15 +30,19 @@ To contribute:
...
 | [Nutonomy](https://www.nutonomy.com/careers/) | Jurong | All-year-around | Autonomous Driving Intern. Note that you need to be on a MS/PhD track |
 | [OpenGovProducts](https://opengovernmentproducts.recruitee.com/)| Funan| All-year-round| SE & UX interns; Only Singaporean or PR allowed;|
+| [Palantir](https://www.palantir.com/careers/)| Various countries (HQ at Palo Alto) | Unknown | Forward deployed engineer has opportunities for travelling |
+| [PayPal](https://www.paypal.com/us/webapps/mpp/jobs)| Suntec City | 6 months | |
```
[the remark this answers is not in the document]
Comment (davidchoo12) [reply README.md]: Ah ok, noted

File: README.md [hunk 3]
```diff
@@ -32,13 +32,16 @@ To contribute:
...
 | [Nutonomy](https://www.nutonomy.com/careers/) | Jurong | All-year-around | Autonomous Driving Intern. Note that you need to be on a MS/PhD track |
 | [OpenGovProducts](https://opengovernmentproducts.recruitee.com/)| Funan| All-year-round| SE & UX interns; Only Singaporean or PR allowed;|
+| [Palantir](https://jobs.lever.co/palantir/?location=Singapore%2C%20Singapore)| Various countries (HQ at Palo Alto) | Unknown | Forward deployed engineer has opportunities for travelling |
+| [PayPal](https://jobsearch.paypal-corp.com/en-US/search?keywords=Engineer%20Intern&location=&facetcountry=sg)| Suntec City | 6 months | |
 | [SEA(Garena)](https://career.seagroup.com/programs?pos=LIP-area)  | One-North | All-year around | |
 | [ShopBack](http://careers.shopback.com/singapore)  | One-North | Summer | No SE intern positions listed |
 | [SEA(Shopee)](https://careers.shopee.sg/) | Kent-Ridge | All-year around | |
 | [Stripe](https://stripe.com/jobs/listing/software-engineering-intern-apac/1806024?gh_src=73vnei) | Outram Park | Summer | |
+| [Titansoft](https://www.titansoft.com/en/career/current-openings?country=singapore) | 90 Eu Tong Sen Street #05-01 | 6 months and summer available | |
```
Comment (sushinoya) [inline README.md]: Sorry for spotting this only later - I think for place, we can just leave it to be `Outram Park` to be consistent with all the other entries in the list. On second thought, it doesn't seem like a very useful column given how small SG is.

**Comment by sushinoya** [comment]
Microsoft and Palantir both don't seem to be offering internships. Adding PayPal and Titansoft in a separate PR

**Pull request closed without merging**